src/pages/Overview/OverviewToolbar.tsx

```diff
@@ -48,6 +44,16 @@ const overviewTypes = {
   service: 'Services'
 };
 
+// TODO Use Object.fromEntries when available
+const sortTypes = (function() {
+  let o = {};
+  Sorts.sortFields.forEach(sortType => {
+    let id: string = sortType.id;
+    Object.assign(o, { [id]: sortType.title });
```

Comment:
Most folks  these days prefer the spread operator over `Object.assign`
https://medium.com/@corinnemariekelly/object-assign-vs-spread-operator-577c889dbadc

Reply:
I may update that going forward.  For now I'll leave as is.  Also, when Object.fromEntries becomes available it will be a 1-liner.